### DISCUSSION TITLE
Video Autoplay/Playlist

### DISCUSSION THREAD

**Discussion by shmaltz** [vidstack/player#1197] [Q&A] [answered]
I am using the cdn to load the video player on a site, and I'm playing YouTube videos. I would like to be able to add an "Autoplay" toggle, where if enabled it will automatically play the next video in the playlist. Is this possible?

**Comment by mihar-22** [comment, marked as answer]
Sure, you would build this into your application. Keep track of the playlist and change the player `src` property when media ends.

**Reply by shmaltz** [reply]
Thank you! I will try that.
